src/index.scss

```diff
@@ -1,6 +1,9 @@
 $spacer: 1rem;
 $blue: #007db8;
 $white: #fff;
+$component-active-bg: #0A3055FF !default;
+$component-active-color: $white !default;
```

Comment:
Easy to follow this one

https://github.com/openedx/paragon/blob/0077397963cdd38462df72071a658317981c926b/scss/core/_variables.scss#L666
```scss
$component-active-color:      $white !default;
```